libyul/AsmAnalysis.h

```diff
@@ -61,10 +57,10 @@ class AsmAnalyzer: public boost::static_visitor<bool>
 	explicit AsmAnalyzer(
 		AsmAnalysisInfo& _analysisInfo,
 		langutil::ErrorReporter& _errorReporter,
-		EVMVersion _evmVersion,
+		dev::solidity::EVMVersion _evmVersion,
```

Comment:
I think this might also move to langutil at some point.

Reply:
it's actually Ethereum/EVM related. Whereas all the other classes (ignoring scanner/token) can be used by *any* compiler frontend. Scanner could even be made generic too. But with EVMVersion, I'd rather place it into libevmasm, tbh.